### ISSUE TITLE
Issue with loading a file to be played

### ISSUE THREAD

**Issue by solidsnake745** [samphonic/MoppyClassic#1]
I'm unable to get the application to load a file to be played. It pops up with a java.lang.NullPointerException every time I try to load the Tetris sample file. The specific line I'm having trouble with is line 248 in MoppyMainWindow.java:

app.ms.loadFile(sequenceFile.getPath());

It has the correct path before execution, but generates the exception anyway. Any suggestions?

**Comment by samphonic** [comment]
Is there any additional stack trace information other than the exception type?  Like maybe in the output window?  That's definitely the last place I thought an error would pop-up.  =P  If there's no additional stack trace info, I can try adding some more debug statements and have you try again.

**Comment by solidsnake745** [comment]
Sorry for such a delayed response. Unfortunately I only get time to do this kind of stuff on the weekends =P

Yeah, I thought it was a weird error too and I hate to admit it, but I had run out of patience when I posted that. Spent the entire day doing research, hooking it up, and playing with different code. Lastly found your project via a post in the Arduino forums. 

After further debugging, it turns out it was all being caused by the serial port not getting initialized. After changing it from the default value of COM6 to my Arduino's port in MoppySequencer, everything worked perfectly! 

It's odd though that after the program initially tries to connect on a preset value rather than the passed in value. Line 33 of MoppySequencer is what I'm talking about specifically.

mb = new MoppyBridge("COM6"); //Create MoppyBridge on the COM port with the Arduino

It's a small fix, but that line should have the variable comPort instead of the string "COM6". Otherwise no matter what you do in the UI, it will just keep trying to connect to COM6.

Thanks for making this! I've got it playing a modified version of the Imperial March that I edited with FruityLoops =D
I'll have to post a video for you somehow.
